e2e/docs/E2E_FRAMEWORK.md

```diff
@@ -1,229 +0,0 @@
-# Pocket 1.0 E2E Testing Framework
```

Comment:
This deleted the entire document without adding a new one.

Reply:
Nvm, @dylanlott showed me whats what offline: https://raw.githubusercontent.com/pokt-network/pocket/main/e2e/docs/E2E_ADR.md